RFCs/0027-Specify-Environment-Variables-In-Runsettings.md

```diff
@@ -0,0 +1,32 @@
+# 0027 Specifying Environment Variables In RunSettings File
+
+## Summary
+Specifying environment variables in the runsettings file. The environment variables can be set which can directly interact with the test host.
+
+## Motivation
+Specifying environment variables in the runsettings file is necessary to support non-trivial projects that require settings env vars like DOTNET_ROOT. These variables are set while spawning the test host process, thus will be available in the host.
+
+## Usage
+The runsettings contains a "EnvironmentVariables" node in the RunConfiguration section.
+The different environment variables can be specified as element name and it's value.
+Below is a sample runsettings for passing environment variables.
+
+```csharp
+
+<?xml version="1.0" encoding="utf-8"?>
+<!-- File name extension must be .runsettings -->
+<RunSettings>
+  <RunConfiguration>
+    <EnvironmentVariables>
```

Comment:
nit: do you usually use plural or singular? I.e. you have `RunSettings` (plural) but also `RunConfiguration` (singular).

Reply:
For the `EnvironmentVariables`, it made sense to use plural to infer that multiple values are allowed.